### PR TITLE
initial skeleton adapting johpapas styleguide

### PR DESCRIPTION
**Pull request by letterix**
closes #51

### CODE REVIEW

File: src/app/components/profile/profile.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('xyz-cv-ui.profile')
+        .controller('Profile', Profile);
+
+        //Profile.$inject = [];
+        function Profile() {
+            /* jshint validthis: true */
```
Comment (Tobbe) [inline src/app/components/profile/profile.js]: This is not needed, right?

File: src/app/blocks/exception/exception.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('blocks.exception')
+        .factory('exception', exception);
+
+    /* @ngInject */
```
Comment (Tobbe) [inline src/app/blocks/exception/exception.js]: Do you need really need this? Doesn't ng-annotate handle this case? You're using a really old version of gulp-ng-annotate, try upgrading to the latest version and see if that helps

File: src/app/components/dashboard/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    'use strict';
+
+    angular
+        .module('xyz-cv-ui.dashboard')
+        .controller('Dashboard', Dashboard);
+
+        //Dashboard.$inject = [];
+
+        function Dashboard() {
+            /* jshint validthis: true */
```
Comment (Tobbe) [inline src/app/components/dashboard/dashboard.js]: Shouldn't be needed (please look for other "jshint" lines to remove as well)

**Comment by letterix** [comment]
hmm, upon removing the jshint lines, i get a lot of "Possible strict violation." errors in jshint

**Comment by Tobbe** [comment]
Yeah, some might still be needed, but please read this (especially the first "Note"): https://github.com/johnpapa/angular-styleguide#style-y032

**Comment by letterix** [comment]
updated gulp-ng-annotate and removed duplicate injection code

**Pull request merged**